src/ontology/cl-edit.owl

```diff
@@ -5242,9 +5243,10 @@ AnnotationAssertion(owl:deprecated obo:CL_0000194 "true"^^xsd:boolean)
 AnnotationAssertion(rdfs:label obo:CL_0000195 "obsolete Purkinje fiber")
 AnnotationAssertion(owl:deprecated obo:CL_0000195 "true"^^xsd:boolean)
 
-# Class: obo:CL_0000196 (flight muscle cell)
+# Class: obo:CL_0000196 (insect flight muscle cell)
 
-AnnotationAssertion(rdfs:label obo:CL_0000196 "flight muscle cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.2142/biophysics.7.21") obo:IAO_0000115 obo:CL_0000196 "A muscle cell of a muscle that is involved in the mechanism of insect flight. This encompasses both cells that power flight and cells that control flight.")
```

Comment:
> Consider adding a comma after "both" (or removing it), as it reads like there are only two cells that power flight.

It would never have occurred to me to interpret that sentence like that. o_O Thanks!